lib/pages/company_site/product_creation_page.rb

```diff
@@ -0,0 +1,311 @@
+
+    # Блок Фото
+    button(:load_image_button, xpath: '//div[@class="image-loader__block"]/../..//input[@type="file"]')
+    image(:first_mini_image, css: 'img.aui-admin-image-preview__image')
+    image(:last_mini_image, xpath: '(//img[@class="aui-admin-image-preview__image"])[last()]')
+    text_area(:description_of_photo, css: '.image-block__textarea .aui-admin-textarea__field')
+    div(:image_error, css: 'div.image-loader__error-text')
+
+    # Блок Группа товаров
+    elements(:groups_tree, css: '.groups__selector-block .aui-admin-rubric-select__title')
+
+    # Блок Документы
+    button(:load_file, xpath: '//div[@class="document-loader__block"]/../..//input[@type="file"]')
+    elements(:docs, css: '.documents__reorder-block .document-block')
+
+    # Блок Сопутствующие товары
+    text_area(:soputka_input, css: '.related-products__search .aui-admin-search__input')
+    button(:soputka_find_button, css: '.related-products__search .aui-admin-search__button')
+    button(:add_in_soputka_button, css: '.related-products__search-products .aui-admin-related-products__button')
+
+    # Минимальный размер заказа (только для ПЦ)
+    text_area(:min_qty_input, css: '.min-quantity .aui-admin-number-input__field')
+
+    # Кнопка сохранения
+    button(:save, css: '.aui-admin-button_submit')
+
+    # Удаление и выход из редактирования
+    # Открывается попап удаления/выхода с одинаковыми классами у обеих кнопок, но разным текстом
+    button(:quit, css: '.product .page-info__edit-product-control-buttons_quit')
+    button(:delete, css: '.product .page-info__edit-product-control-buttons_delete')
+    span(:edit_popup, css: '.product .page-info-popup__block')
+    button(:edit_yes, css: '.page-info-popup__block [type="submit"]')
+    button(:edit_no, css: '.page-info-popup__block .page-info-popup__link')
+
+    # Модерация
+    #   Кнопки на форме
+    button(:accept, css: '.aui-admin-moderation-button_accept')
+    button(:decline, css: '.aui-admin-moderation-button_decline')
+    button(:postpone, css: '.aui-admin-moderation-button_postpone')
+    #   Попап модерации
+    span(:moderation_popup, css: '.moderation-popup__block')
+    textarea(:cause_input, css: '.moderation-popup__block .moderation-popup__textarea')
+    select(:cause_select, css: '.moderation-popup__block .aui-admin-select__input')
+    button(:moderation_submit, css: '.moderation-popup__block .aui-admin-button')
+    #   Уведомление
+    span(:moderation_message, css: 'div.page-info__notice')
+
+    # Батарейка
+    span(:thermometer_degree, css: '.aui-admin-battery__percent')
+    link(:thermometer_degree_popup_link, css: '.aui-admin-battery__link')
+    span(:thermometer_degree_popup, css: '.aui-admin-popup')
+
+    # ОПИСАНИЕ МЕТОДОВ
+    #
+    # Основной метод заполнения атрибутов, которые переданы в переменную options спека
+    # Если атрибут есть в переменной options, то заполняется методом
+    # Если атрибута нет, то пропускается
+    def fill_attributes(options)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for fill_attributes is too high. [<7, 46, 19> 50.26/35]<br>Metrics/CyclomaticComplexity: Cyclomatic complexity for fill_attributes is too high. [20/7]<br>Metrics/PerceivedComplexity: Perceived complexity for fill_attributes is too high. [20/8]